### PR TITLE
Add current frame to the manual frames hotkey

### LINKED ISSUE #75: Add a hotkey to paste current frame into Manual Frame row
It's quite tedious copying, pasting, putting a comma and a space for every frame I wanna add to my comp. A hotkey that takes the current frame and adds it to the manual list of frames would be amazing.
image

### PR DESCRIPTION
**Pull request by Kapppa**
Closes #75

### CODE REVIEW

File: vspreview/toolbars/misc/toolbar.py [hunk 1]
```diff
@@ -171,6 +171,25 @@ def add_shortcuts(self) -> None:
                 ViewMode.FFTSPECTRUM if self.main.current_viewmode != ViewMode.FFTSPECTRUM else ViewMode.NORMAL
             )
         )
+        self.main.add_shortcut(
+            Qt.Key.Key_Return, self.add_current_frame_to_comp
```
Comment (Setsugennoao) [inline vspreview/toolbars/misc/toolbar.py:175]: Return key seems strange, I'd use the same keybind as the scening toolbar

File: vspreview/toolbars/misc/toolbar.py [hunk 2]
```diff
@@ -171,6 +171,25 @@ def add_shortcuts(self) -> None:
...
+        )
+
+    def add_current_frame_to_comp(self) -> None:
```
Comment (Setsugennoao) [inline vspreview/toolbars/misc/toolbar.py:178]: this should be in the comp toolbar

File: vspreview/toolbars/misc/toolbar.py [hunk 3]
```diff
@@ -171,6 +171,25 @@ def add_shortcuts(self) -> None:
...
+        comp = self.main.toolbars.comp
+        if not comp.manual_frames_lineedit.isVisible():
```
Comment (Setsugennoao) [inline vspreview/toolbars/misc/toolbar.py:180]: if in the comp toolbar, this will be unnecessary

File: vspreview/toolbars/misc/toolbar.py [hunk 4]
```diff
@@ -171,6 +171,25 @@ def add_shortcuts(self) -> None:
...
+            return
+
+        frame = str(self.main.current_output.last_showed_frame)
+        current_frames = comp.manual_frames_lineedit.text()
```
Comment (Setsugennoao) [inline vspreview/toolbars/misc/toolbar.py:184]: add strip

File: vspreview/toolbars/misc/toolbar.py [hunk 5]
```diff
@@ -171,6 +171,25 @@ def add_shortcuts(self) -> None:
...
+
+        if not current_frames:
```
Comment (Setsugennoao) [inline vspreview/toolbars/misc/toolbar.py:186]: this check isn't needed, code below will generate an empty list and will just work

File: vspreview/toolbars/misc/toolbar.py [hunk 6]
```diff
@@ -171,6 +171,25 @@ def add_shortcuts(self) -> None:
...
+            comp.manual_frames_lineedit.setText(frame)
+        else:
+            current_frames = current_frames.split(",")
```
Comment (Setsugennoao) [inline vspreview/toolbars/misc/toolbar.py:189]: i'd also make this a set, maybe reorder too?

**Review by Setsugennoao — CHANGES_REQUESTED** [review]

File: vspreview/toolbars/misc/toolbar.py (hunk 5 above)
Comment (Kapppa) [reply vspreview/toolbars/misc/toolbar.py:186]: No it won't 

```py
>>> "".split(",")
['']
```

**Pull request merged**